tox_lin.ini

```diff
@@ -0,0 +1,19 @@
+[tox]
+minversion = 4.0.0
+env_list = py{310,311}
+isolated_build = true
+
+[gh-actions]
+python =
+    3.10: py310
+    3.11: py311
+
+[testenv]
```

Comment:
Let's add flake8 and mypy here
<img width="492" alt="image" src="https://github.com/user-attachments/assets/07c9816a-5d99-4eba-a310-993f6f1245fc" />

Reply:
Added